auto/config

```diff
@@ -6,22 +6,18 @@ lb config noauto \
 	--linux-packages linux-image-rt \
 	--distribution bookworm \
 	--binary-images iso-hybrid \
-	--debian-installer netboot \
-	--archive-areas "main contrib non-free non-free-firmware" \
```

Comment:
What is the point of this change?  It seem like a non-change to me, reordering non-free and non-free-firmware?

Reply:
I need to revisit this as there is some preseed script that overrides this and did not include non-free-firmware. I thought I deleted it. It may still be active based on recent tests. We need non-free because some network drivers such as the R8168-dkms are not firmware so remain in non-free.

Reply:
It appears I  had a regression here. and with the preseed